cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/templates/etl/batch/ETLStreamConversionTest.java

```diff
@@ -25,6 +25,7 @@
 import co.cask.cdap.templates.etl.batch.config.ETLBatchConfig;
 import co.cask.cdap.templates.etl.batch.sinks.TimePartitionedFileSetDatasetAvroSink;
 import co.cask.cdap.templates.etl.batch.sources.StreamBatchSource;
+import co.cask.cdap.templates.etl.common.MockAdapterConfigurer;
```

Comment:
This is required since I moved this class to etl-common since it is needed in both etl-realtime and etl-batch.